core/storage/suggestion/gae_models_test.py

```diff
@@ -1283,6 +1283,33 @@ def test_get__suggestions_waiting_too_long_raises_if_suggestion_types_empty(
                     .get_suggestions_waiting_too_long_for_review()
                 )
 
+    def test_get_new_suggestions_waiting_for_review(self) -> None:
+        suggestion_type = feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT
+        max_suggestions = 1
+        creation_time = datetime.datetime(2020, 6, 14, 5)
```

Comment:
make it timezone aware

Reply:
I think the creation time will be in UTC only as I am mocking it here https://github.com/oppia/oppia/pull/19045/files#diff-1aefc101e4fc62f2d786e1863f699b6ee3bc7f4ed879a36a989f9488c864a1baR1298